tests/test_module.py

```diff
@@ -43,7 +43,7 @@ def test_train(model_name, use_atomref, tmpdir):
     prior = None
     if use_atomref:
         prior = getattr(priors, args["prior_model"])(dataset=datamodule.dataset)
-        args["prior_args"] = prior.get_init_args()
+        args["prior_init_args"] = prior.get_init_args()
```

Comment:
The loading of the pretrained models (https://github.com/torchmd/torchmd-net/tree/main/examples#loading-checkpoints) fails:

```python
from torchmdnet.models.model import load_model
load_model('ANI1-equivariant_transformer/epoch=359-val_loss=0.0004-test_loss=0.0120.ckpt')

AssertionError: Requested prior model Atomref but the arguments are lacking the key "prior_init_args".
```

Reply:
It sounds like we want to redo how prior args are specified as described in https://github.com/torchmd/torchmd-net/issues/26#issuecomment-1299237398.  That means we can switch back to `prior_args` for this value.  There will still be compatibility issues, because it will need to become a list with args for multiple prior models, but I can add a check for that case for backward compatibility.  I'll go ahead and make the changes in this PR.

Reply:
Might be a good idea to add a test case for loading model checkpoints from a previous version.